ceph.spec.in

```diff
@@ -2522,6 +2535,14 @@ fi
 
 %postun -n libcephfs2 -p /sbin/ldconfig
 
+%files -n libcephfs-proxy0
+%{_libdir}/libcephfs_proxy.so.*
+%{_bindir}/libcephfsd
```

Comment:
The binary is found in the sbin directory.  This line should be
%{_sbindir}/libcephfsd

instead

Reply:
Done